docs/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md

```diff
@@ -13,66 +13,59 @@ The `./config/database.js` file is used to define database connections that will
+  - `client` (string): Database client to create the connection. `sqlite` or `postgres` or `mysql`.
+  - `connection` (object): Database connection information
+    - `host` (string): Database host name. Default value: `localhost`.
+    - `port` (integer): Database port.
+    - `database` (string): Database name.
+    - `username` (string): Username used to establish the connection.
+    - `password` (string): Password used to establish the connection.
+    - `timezone` (string): Set the default behavior for local time. Default value: `utc` [Timezone options](https://www.php.net/manual/en/timezones.php).
+    - `schema` (string): Set the default database schema. **Used only for Postgres DB.**
+    - `ssl` (boolean/object): For ssl database connection. Object is used to pass certificate files as strings.
+  - `useNullAsDefault` (boolean): Will use `NULL` as a default value **Used only for SQLite**
+  - `debug` (boolean): Show database exchanges and errors.
+    <!-- - `autoMigration` (boolean): To disable auto tables/columns creation for SQL database. -->
+  - `pool` (object _optional_): [Tarn.js](https://github.com/vincit/tarn.js) database pooling options
+    - `min` (integer): Minimum number of database connections to keepalive Default value: `0`
+    - `max` (integer): Maximum number of database connections to keepalive Default value: `10`
+    - `acquireTimeoutMillis` (integer): Time in ms before timing out a database connection attempt
+    - `createTimeoutMillis` (integer): Time in ms before timing out a create query attempt
+    - `destroyTimeoutMillis` (integer): Time in ms before timing out a destroy query attempt
+    - `idleTimeoutMillis` (integer): Time in ms before free database connections are destroyed
+    - `reapIntervalMillis` (integer): Time in ms to check for idle database connections to destroy
+    - `createRetryIntervalMillis` (integer): Time in ms to idle before retrying failed create actions
+    - `afterCreate` (function): Callback function to execute custom logic when the pool acquires a new connection. See the [Knex.js documentation](https://knexjs.org/#Installation-pooling) for more information
+
+<!-- TODO: Open and track a feature request for autoMigration as it doesn't exist in v4 -->
+
+### Configuration Examples
 
 ::::: tabs card
 
 :::: tab PostgreSQL
 
 ```js
 module.exports = ({ env }) => ({
-  defaultConnection: 'default',
-  connections: {
-    default: {
-      connector: 'bookshelf',
-      settings: {
-        client: 'postgres',
-        host: env('DATABASE_HOST', 'localhost'),
-        port: env.int('DATABASE_PORT', 5432),
-        database: env('DATABASE_NAME', 'strapi'),
-        username: env('DATABASE_USERNAME', 'strapi'),
-        password: env('DATABASE_PASSWORD', 'strapi'),
-        schema: env('DATABASE_SCHEMA', 'public'), // Not Required
-        ssl: {
-          rejectUnauthorized: env.bool('DATABASE_SSL_SELF', false), // For self-signed certificates
-        },
+  connection: {
```

Comment:
suggestion: We now add the path to file in a comment in JS code snippets.

So I'd change this for:

```js
// path: ./config/database.js

module.exports…
```